### ISSUE TITLE
rutorrent can not load magnet links

### ISSUE THREAD

**Issue by alexfsoko** [Novik/ruTorrent#1473]
Hi, hope you can help me.

When I add a magnet link through rutorrent, nothing happens. No file appears at download list.
Torrents are loaded and downloaded without problem.
I follow this guide:
https://blog.quad.moe/setting-up-a-rutorrent-seedbox/
 I have a seedbox in scaleway / ubuntu denial

[10.06.2017 12:42:39] WebUI started.
[10.06.2017 12:42:43] rss: Some functionality will be unavailable. Webserver user can't access external program (curl).
[10.06.2017 12:42:43] rutracker_check: Plugin will not work. Webserver user must have execute access to the rtorrent session directory (/root/.session/).
[10.06.2017 12:42:43] mediainfo: Plugin will not work. rTorrent user can't access external program (mediainfo).
[10.06.2017 12:42:43] screenshots: Plugin will not work. rTorrent user can't access external program (ffmpeg).
[10.06.2017 12:42:46] Unpack plugin: rTorrent user can't access 'unzip' program.
[10.06.2017 12:42:46] Unpack plugin: rTorrent user can't access 'unrar' program.

root@xxxxxxx:/var/www# ls -l rutorrent/
total 56
drwxrwxr--  3 root www-data  4096 Jun 10 18:25 conf
drwxrwxr--  2 root www-data  4096 Jun 10 11:53 css
drwxrwxr--  2 root www-data  4096 Jun 10 11:53 images
-rwxrwxr--  1 root www-data 14009 Jun 10 11:53 index.html
drwxrwxr--  2 root www-data  4096 Jun 10 11:53 js
drwxrwxr--  2 root www-data  4096 Jun 10 11:53 lang
-rwxrwxr--  1 root www-data   670 Jun 10 11:53 LICENSE.md
drwxrwxr--  2 root www-data  4096 Jun 10 11:53 php
drwxrwxr-- 46 root www-data  4096 Jun 10 11:53 plugins
-rwxrwxr--  1 root www-data  1683 Jun 10 11:53 README.md
drwxrwxr--  5 root www-data  4096 Jun 10 11:53 share

**Comment by ChriZathens** [comment]
Have you fixed your error?

**Comment by Pingblue** [comment]
Looks like an install problem.

Have you tried installing the required programs?

apt-get install mediainfo unzip unrar ffmpeg curl

Your also running rtorrent as the root user.

**Comment by alexfsoko** [comment]
I've fixed the problem, very simple...
After setup rutorrent, my first torrent was a magnet link without tracker.... I can not download anything, all icons in red. Then, I've downloaded a torrent from a known tracker (not magnet, torrent file) and starts great... and then I've restarted the torrents from magnet links an voila! It got the peers and the torrent started! I think they piggiback the tracker from the torrent file to discover the DHT nodes and connect to the peers...
